core/src/main/resources/reference.conf

```diff
@@ -22,8 +22,14 @@ akka {
         illegal-header-warnings = off
       }
     }
+    server {
+      # By default, akka includes an HTTP header in responses that looks like:
+      # server=akka-http/10.20.1
+      # For better app sec, we suppress this to make it harder for attackers to learn about our system.
+      # Akka doc: https://doc.akka.io/docs/akka-http/current/configuration.html
+      server-header = ""
+    }
```

Comment:
I think the change is great as-is and clearly addresses the appsec report. 👍 

I do wonder whether we could improve it for our own ergonomics by returning e.g. `server=cromwell/87-db0dd2d` so that it's easy to tell you're talking to a Cromwell (as opposed to CromIAM) as well as its version.

Presumably this is not an appsec issue in the same way, seeing as Cromwell version is very much not-secret: https://api.firecloud.org/#/Version/executionEngineVersion